wgpu-core/src/command/compute.rs

```diff
@@ -228,6 +235,14 @@ impl State {
     }
 }
 
+#[repr(packed)]
```

Comment:
I mean none of these make practically a difference here. But I think repr(c) would technically be incorrect. Let's say I have some esoteric architecture where all these 32 bit values have 64bit padding. Then this would violate the WebGPU spec which says the values are all tightly packed, right? Problem is that we're not really after having a usable struct but after a description of the spec

Reply:
I mean an arch where primatives had size != alignment would cause so many issues that it honestly isn't worth thinking about. Either way I think rust guarantees the alignment of primatives. The reason I don't like packed is that it's super easy to get off into _real fuck your shit up_ ub land if you end up with misaligned references and it's use should be discouraged unless there's no way around it.